src/test/data/JsonSerializableAddressBookTest/duplicatePersonAddressBook.json

```diff
@@ -3,25 +3,15 @@
     "name": "Alice Pauline",
     "phone": "94351253",
     "email": "alice@example.com",
-    "personEvents": [{
-      "name": "IFG"
-    }],
-    "roles": [ "friends" ]
+    "eventRoles": {
+      "IFG": ["friends"]
```

Comment:
It is nice to see that you updated the JSON format for the Addressbook to account for the new structure!